cli/translator.js

```diff
@@ -0,0 +1,58 @@
+import fs from 'fs/promises'
+
+export default async function translator(path) {
+  const cwd = process.cwd()
```

Comment:
Translator is not allowed to look at `cwd`. Only the argument given as parameter should be used.

This will make code more predictable, and tests simpler.

Reply:
DONE